keyboards/dumbpad/rules.mk

```
@@ -35,3 +35,5 @@ HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs
 ENCODER_ENABLE = yes
 MOUSEKEY_ENABLE = yes
 KEY_LOCK_ENABLE = yes
+
+DEFAULT_FOLDER = dumbpad/v1x
```

The `v0x` version seems to be the device supported by the code this PR deprecates.

Behaviour should be maintained for existing users, with the goal to remove this rule (and thus require a version specified when using `make`/`qmk compile`) down the road.

```suggestion
DEFAULT_FOLDER = dumbpad/v0x
```

---

Yes, v0x was the original design - it's technically archived in the board repo but I understand leaving it as the default for anyone who is still using v0x.